docs/chart_repository.md

```diff
@@ -270,8 +275,8 @@ fantastic-charts    https://fantastic-charts.storage.googleapis.com
 If the charts are backed by HTTP basic authentication, you can also supply the
 username and password here:
 
-```console
-$ helm repo add fantastic-charts https://username:password@fantastic-charts.storage.googleapis.com
+``console
+$ helm repo add fantastic-charts https://fantastic-charts.storage.googleapis.com --username my-username--password my-password
```

Comment:
missing whitespace between --password and the value supplied to --username.

Reply:
Thanks for noticing this. Fixed.